common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -240,6 +241,7 @@ const SearchTabs: FunctionComponent<Props> = ({
             sortOrder={undefined}
             linkResolver={params => {
               const queryWithSource = propsToQuery(params);
+              /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
               const { source = undefined, ...queryWithoutSource } = {
```

Comment:
for these two I've replicated the "fix" we agreed on in Search (e.g. line 80 of `catalogue/webapp/pages/search/works.tsx` ), we know why we're doing this and are ok with it 👍